src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -88,6 +88,8 @@ type Agency implements Node {
     """
     types: [AgencyAlertType]
   ): [Alert]
+  "URL to the logo of the agency"
+  brandingUrl: String
```

Comment:
I know that I has been imported for quite a while but I cannot actually find this in the GTFS spec. Can you point me towards it?

If it's not in the specification, we would not like to expose in the API and instead remove it from the data model.

If you're keen to use it, can you open a PR or issue on the GTFS spec repo?

Reply:
No, I didn't find it in the GTFS spec but in the onebusaway model as optional so I thought it could be added also in the Graphql schema also as optional.

Reply:
So there is no way to add brandingUrl aside from opening a Issue or PR to the GTFS spec repo?

Reply:
We really would like to no expose anything that isn't in the spec. If you want to discuss this, you can come to the dev meeting.

Reply:
It was a mistake to have it in OBA and OTP in the first place.

Reply:
Ok, thanks. I'll mark this PR as closed then